### DISCUSSION TITLE
Two questions about Parallel module

### DISCUSSION THREAD

**Discussion by wuliwei** [peridigm/peridigm#267] [General]
Dear all,

Now, I have two questions about Peridigm and hope you could give me some presentations. 
1: Where is the parallel module being implemented in Peridigm? I cannot find it.
2: I tried to use the stress or strain criterion in NOS-PD to determine whether the bond is broken or not, but found that the fracture is not uniform. This problem won't happen when I turn parallel off. However, the critical strech criterion has no similar problem like it, which is really stable. 

Thanks,
Liwei

**Comment by oldninja** [comment]
Hi @wuliwei ,
1) You will not find a "parallel module" in Peridigm. Peridigm is a component software project built mainly upon Sandia's Trilinos project and other software components efforts. See: https://www.osti.gov/servlets/purl/1055619
2) Could you please clarify what problem you are trying to solve? Also, what are you expecting to see? What do you mean by the fracture not being uniform?
Thanks!

**Reply by wuliwei** [reply]
Hi oldninja, 

Thanks for your reply.
I tried to use the critical stress criterion in Peridigm. This criterion works well for single-core computing, but for multi-core computing, some cracks propagate that should not exist. I think that parallel computing affects the results. So i wanna know whether something is ignored in my work. 

Best,
Liwei

**Reply by jritta** [reply]
Hello Liwei,

Probably your problem is similar as #119. Have a look there and the implemented solution.

Best,
Jonas
